config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -113,11 +113,16 @@ teams:
     - luxas
     - roberthbailey
     - timothysc
+    - vincepri
     privacy: closed
   cluster-api-provider-gcp-maintainers:
     description: write access to cluster-api-provider-gcp-maintainers
     members:
+    - detiber
     - justinsb
+    - krousey
```

Comment:
I haven't seen kris around in O(year+)

Reply:
I can update if necessary, this is copied from the repo's maintainer list